site/docs/spec.md

```diff
@@ -212,6 +212,24 @@ Columns in Iceberg data files are selected by field id. The table schema's colum
 
 For example, a file may be written with schema `1: a int, 2: b string, 3: c double` and read using projection schema `3: measurement, 2: name, 4: a`. This must select file columns `c` (renamed to `measurement`), `b` (now called `name`), and a column of `null` values called `a`; in that order.
 
+Tables may also define a property `schema.name-mapping.default` with a JSON `name mapping` containing a list of `field mapping` objects. These mappings provide fallback field ids to be used when a data file does not contain field id information. Each object should contain
```

Comment:
`name mapping` and `field mapping` are fixed width, but aren't properties? Why used fixed width font?

Reply:
I thought about this as names of new terms we are defining. We can remove the font

Reply:
Yeah, I think we should.